source/views/menus/components/fancy-menu.js

```diff
@@ -157,23 +158,26 @@ class FancyMenuView extends React.PureComponent {
           filters={filters}
           onPress={this.openFilterView}
         />
-        {messageView
-          ? messageView
-          : <SectionList
-              ItemSeparatorComponent={CustomSeparator}
-              ListEmptyComponent={messageView}
-              keyExtractor={this.keyExtractor}
-              style={styles.inner}
-              sections={grouped}
-              renderSectionHeader={this.renderSectionHeader}
-              renderItem={({item}: {item: MenuItemType}) =>
-                <FoodItemRow
-                  data={item}
-                  corIcons={this.props.menuCorIcons}
-                  badgeSpecials={!specialsFilterEnabled}
-                  spacing={{left: leftSideSpacing}}
-                />}
-            />}
+        {messageView ? (
+          messageView
+        ) : (
```

Comment:
This whole messageview thing feels like it should have its own check before we do the normal-path render